### ISSUE TITLE
Use a different folder containing manifests

### ISSUE THREAD

**Issue by Robert-LTH** [aaronparker/Evergreen#81]
**Is your feature request related to a problem? Please describe.**
When the name in the manifest does not match the name which my organisation has selected to use as the name for an application i need to edit the manifest to change the name. I would rather not edit the files contained in the module.

**Describe the solution you'd like**
I would like to have some way of setting which path to use when looking for manifests

**Describe alternatives you've considered**
Add a setting to Evergreen.json named ManifestsPath(s) and the ability to set this or add another path with a cmdlet

Edit the manifests and holding a copy of the files, when Evergreen is installed (or updated) replace the manifests in the modules folder with the edited ones.

Submit an edited manifest with the new name.

**Additional context**
One example of this is Zoom. In my organisation it is called Zoom Client for Meetings and in the Evergreen manifest it is simply named Zoom.

**Comment by aaronparker** [comment]
I don't understand how this affects scripts etc., that are using Evergreen to download Zoom. Could you please provide more detail. `Get-Zoom` returns the Zoom Client for Meetings as well as other Zoom clients and plug-ins

**Comment by Robert-LTH** [comment]
I came into this with the wrong idea of how to use Evergreen. I am trying to replace a webcrawler with this wonderful project and in that crawler the application consisted of a configuration and a single script which was used for all configurations. I learned that evergreen has a script and a configuration file per application and realised that my request would be to have a custom repository for both the scripts and configuration files. This is essentially what you get if you install the module into your own modules folder as a user.

I think i asked this question a little bit too soon and this "issue" can be closed.


Thank you for a useful and good project!
